src/powerbox/tools.py

```diff
@@ -452,6 +453,11 @@ def get_power(
         If set to False, uses numpy's FFT routine. If set to None, uses pyFFTW with
         number of threads equal to the number of available CPUs. If int, uses pyFFTW
         with number of threads equal to the input value.
+    k_factor_fnc : callable, optional
+        A function that takes in an array of wavemodes [kx, ky, kz] and returns an array
+        of the same size. This function is applied on the FT before the angular averaging.
+        It can be used, for example, to change units from power to another
```

Comment:
Add a little more docs here to make it clear that this can be used for converting linearly-binned power into power-per-logarithmic k (Delta^2). Then say "see examples for more info" and write an Examples section.